lib/screens/add_recipe_screen.dart

```diff
@@ -16,12 +13,38 @@ class _AddRecipeScreenState extends State<AddRecipeScreen> {
   final _nameController = TextEditingController();
   final _imageUrlController = TextEditingController();
   final _descriptionController = TextEditingController();
+  final _instructionsController = TextEditingController();
+
+  String? _validate(String value, String errorText) {
+    if (value.isEmpty) {
+      return errorText;
+    }
+    return null;
+  }
+
+  String? _validateUrl(String value, String errorText) {
+    if (value.isEmpty ||
+        (!value.startsWith('http://') && !value.startsWith('https://'))) {
+      return errorText;
+    }
+    return null;
+  }
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: Colors.black,
       appBar: AppBar(
-        title: const Text('Add Recipe'),
+        backgroundColor: context.colorScheme.primary,
+        title: Text(
+          'ADD RECIPE',
+          style: TextStyle(
```

Comment:
you can set up it in the Theme and use as textTheme

Reply:
the textStyle you mean, it is repeated a lot maybe it can be done a a separate widget

Reply:
about textTheme I do not know much, maybe explain tomorrow